src/ui/actions/app.ts

```diff
@@ -202,6 +203,56 @@ export function loadMouseTargets(): UIThunkAction {
   };
 }
 
+export function setBreakpointsFromClipboard(): UIThunkAction {
+  return async (dispatch, getState, { ThreadFront }) => {
+    const currentFocusedElement = document.activeElement;
+    let text = "";
+    try {
+      // The `readText()` API requires that the current DOM be focused somehow
+      document.body.focus();
+      text = await navigator.clipboard.readText();
+
+      // Was the clipboard data even valid JSON?
+      const parsedValue = JSON.parse(text);
+      // Assuming it was JSON, does it appear to be an array of Point objects?
+      if (Array.isArray(parsedValue) && parsedValue.every(isValidPoint)) {
+        // TODO [FE-874] This will need to change if we modify how we do per-recording persistence
+
+        // This value appears to be actual breakpoints. Our current implementation watches
+        //  for changes in `localStorage` and will auto-update itself when that event fires.
+        const storageKey = `${ThreadFront.recordingId!}::points`;
```

Comment:
nit: Would be nice to have a single function for this:
```js
function getLocaleStorageKeyForPoints(recordingId: RecordingId): string {
  return `${recordingId}::points::transition`;
}
```

That way if we ever change one, we'll _know_ about the other one.